recipes/angsd/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "0.910" %}
+
+build:
+  number: 0
+  skip: True # [osx]
+
+about:
+  home: http://www.popgen.dk/angsd/index.php/ANGSD
+  license: GPLv3, MIT
+  summary: "ANGSD: Analysis of next generation Sequencing Data"
+
+package:
+  name: angsd
+  version: {{ version }}
+
+requirements:
+  build:
+    - htslib
```

Comment:
You should add gcc/llvm as described in our Guideline to the build requirements and libggc as runtime dep.